tests/test_registrations/test_retractions.py

```diff
@@ -28,11 +28,18 @@
 class RegistrationRetractionModelsTestCase(OsfTestCase):
     def setUp(self):
         super(RegistrationRetractionModelsTestCase, self).setUp()
+        self.mock_registration_update = mock.patch('website.project.tasks.on_registration_updated')
```

Comment:
And you've verified that these tests function properly, yes? (I ask this of any tests that include mocks now, since we've had a bit of a history with mocking tests not actually doing anything).